### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by stevage**

### CODE REVIEW

**Comment by stevage** [comment]
Actually not that sure. Maybe it should be going to https://terria.io/Documentation/guide/contributing/ (ie, a full URL not a relative one). Does this page get rendered somewhere other than Github itself?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Reviewers are welcome to make minor edits to a pull request (e.g. fixing typos)
 
 After you merge a pull request, delete the associated branch if you can.  You will see an option for this on the GitHub pull request page.
 
-More information in the [Contributor's Guide](Documentation/Contributors)
+More information in the [Contributor's Guide](CONTRIBUTORS.md)
```
Comment (kring) [inline CONTRIBUTING.md]: CONTRIBUTORS.md is just the list of people who have contributed. The best guide is probably this one:
https://terria.io/Documentation/guide/contributing/

**Comment by kring** [comment]
No, CONTRIBUTING.md is only used on github.

**Comment by stevage** [comment]
ok, updated.

**Pull request merged**